support-frontend/assets/helpers/subscriptionsForms/checkoutFormIsSubmittableActions.ts

```diff
@@ -15,18 +16,15 @@ const enableOrDisablePayPalExpressCheckoutButton = (
 };
 
 function enableOrDisableForm() {
-	return (
-		dispatch: (...args: any[]) => any,
-		getState: () => CheckoutState,
-	): void => {
+	return (_dispatch: Dispatch, getState: () => CheckoutState): void => {
 		enableOrDisablePayPalExpressCheckoutButton(checkoutFormIsValid(getState()));
 	};
 }
 
 function setFormSubmissionDependentValue(setStateValue: () => Action) {
-	return (dispatch: (...args: any[]) => any): void => {
+	return (dispatch: Dispatch<Action>, getState: () => CheckoutState): void => {
 		dispatch(setStateValue());
-		dispatch(enableOrDisableForm());
+		enableOrDisableForm()(dispatch, getState);
```

Comment:
was this a bug before?

Reply:
Kind of, it did work because of some Redux magic or other but the types never lined up. Tbh this code is all pretty nasty but at least it makes sense now if you follow it through.